src/views/List.jsx

```diff
@@ -8,7 +8,7 @@ export function List({ data }) {
 			</p>
 			<ul>
 				{data.map((item, index) => {
-					return <ListItem key={index} name={item.id} />;
+					return <ListItem key={index} name={item.name} />;
```

Comment:
I wasn't able to review previous PRs but one small suggestion here is when using a map, it's more effective for the key to be the ID of the item instead of the index in the map. If a key is not specified, React automatically uses the index anyways so it can be redundant to define a key like that. You can learn more about it in the [React Docs](https://react.dev/learn/rendering-lists#why-does-react-need-keys)

Reply:
That makes sense! Thanks for the direct link, another pitfall mentioned on that page is 'don't use Math.random() to generate keys, as the element would then be recreated and form data would be lost on that next render` which is something I had never even considered.

I'll make that change and add it to this PR.